joomla/joomla-cms
# Solution for tagging in different languages

Pull Request for Issue # .
### Summary of Changes

A solution for: https://github.com/joomla/joomla-cms/issues/10761
The issue was created by "event.which === 188"  which means ',' on keyup event but I changed it to 2 different events. Keyup to check if enter is pressed and keypress to check if ',' is pressed while typing in English. An event listener is changed to two event listener and a function.
### Testing Instructions

I tested code with multiple languages such as Hebrew, Persian, Arabic and ... to see if there is a key charCode equal to 44 among their numbers and alphabet which would cause issues and other key code tests and typing test in Joomla tag field to make sure there is no conflict. I found out that 44 is probably an unique charCode. The code passed all tests successfully
### Documentation Changes Required

There is an issue in typing tags in different languages. Pressing ',' in tags means you are done and it will be saved, but it's only for English not other languages such as Arabic, Persian, Hebrew and ... which was bothering me and many other Joomla users. So that I made changes to solve this problem. Now, you can press ',' but it submits tag just if you are typing in English language and if you are typing in other languages continue as long as you want and press enter at the end. Tested in many languages and everything works fine.
